packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -376,7 +404,9 @@ You can't publish integrations with the following Okta features in the OIN catal
   * Custom scopes, such as the `groups` scope, aren't supported for integrations published in the OIN.
   * ISVs shouldn't rely on the `email_verified` scope-dependent claim returned by an OIDC integration to evaluate whether a user has verified ownership of the email address associated with their profile.
 
-* **SAML encryption:** SAML integrations must use SHA256 encryption for security. If you're using SHA-1 for encryption, see our guide on how to [Upgrade SAML Apps to SHA256](/docs/guides/updating-saml-cert/).
+* **SHA-1 SAML encryption:** SAML integrations must use SHA256 encryption for security. If you're using SHA-1 for encryption, see our guide on how to [Upgrade SAML Apps to SHA256](/docs/guides/updating-saml-cert/).
```

Comment:
I don't think these need to be in bold font, which is contrary to our style guide. I'd remove this for all entries in this section.

I'd almost say these bullet "headings" are unnecessary

Reply:
I agree, this doesn't follow our style.
But I think this was recommended by the SME/support to highlight the features not supported (since we had a wall of text on the page).